wire/net/simple/listener_internal_test.go

```diff
@@ -80,7 +80,7 @@ func TestListener_Accept(t *testing.T) {
 		defer l.Close()
 
 		test.AssertNotTerminates(t, timeout, func() {
-			l.Accept()
+			_, _ = l.Accept()
```

Comment:
Do something with the return values or use `//nolint`?

Reply:
See discussion with the `_ =` above.

Reply:
See answer above.

Reply:
Already updated it in the last iteration.